financial-connections/src/main/java/com/stripe/android/financialconnections/repository/FinancialConnectionsManifestRepository.kt

```diff
@@ -120,6 +120,8 @@ internal interface FinancialConnectionsManifestRepository {
         sessionId: String
     ): FinancialConnectionsAuthorizationSession
 
+    suspend fun pollAccountNumbers(linkedAccounts: Set<String>)
```

Comment:
this should probably live in `FinancialConnectionsAccountsRepository` - originally tried to break the repositories down by responsibility but it's a bit messy now 🙈

Reply:
Moved it!